src/grants/refresh_token.grant.ts

```diff
@@ -79,32 +79,4 @@ export class RefreshTokenGrant extends AbstractGrant {
 
     return refreshToken;
   }
-
```

Comment:
moved to client_credentials grant. So technically here the breaking change is that the server needs client_credentials grant to be enabled to use the refresh_token grant, whereas before it needed to enable the refresh_token. I might reconsider this, but just noting it for now

Reply:
Not a problem if the server enables client credentials and refresh token grant by default